package.json

```diff
@@ -117,7 +117,7 @@
     "@solana/kit": "^2.1.0",
     "@solana/web3.js": "^1.31.0",
     "@types/mocha": "^10.0.1",
-    "@uma/sdk": "^0.34.10",
+    "@uma/contracts-node": "^0.4.26",
```

Comment:
nb. We can ultimately propose to relocate the `poolClient` implementation to the fe, which is afaik the only place it's actually used. At that point we'd then be able to drop this dependency as well.